packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -221,11 +215,28 @@ class TimeInput extends React.Component {
       }
     }
 
+    let skipStateUpdate = true;
+    // Data Attribute `data-date-time-input` of date-time-picker is used to ensure state update happens only when props are updated in date-time-picker.
```

Comment:
What does this comment mean? How is this used to ensure state update happens only when props are updated in `date-time-picker`?

Reply:
this changes has been removed : 94624440855ecca7249f9c3cd35ae80cda292300